Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest/Contest.tsx

```diff
@@ -22,7 +23,7 @@ const Contest = () => {
             score,
             maxScore,
             remainingTimeInMilliseconds,
-            validationResult,
+            validationResult: contestValidation,
```

Comment:
Is this needed with the new changes?

Reply:
No, it is not, but I am removing it in another PR, where I am handling the contest validation. This one is only for the problem results :D